pkg/clusteragent/orchestrator/status.go

```diff
@@ -101,16 +94,12 @@ func GetStatus(apiCl kubernetes.Interface) map[string]interface{} {
 		status["LeaderError"] = err
 	} else {
 		status["Leader"] = engine.IsLeader()
-		if ip, err := engine.GetLeaderIP(); err == nil {
-			status["LeaderIP"] = ip
-		} else {
-			status["LeaderError"] = err
-		}
+		status["LeaderName"] = engine.GetLeader()
```

Comment:
Do we want to handle the empty leader case? Example we are in the process of electing. Else this line would be `" "`. 
Instead of showing an empty string we could for now show `no leader detected` or so and maybe (in a future pr?) redirect to another section which has more information why there is no leader currently.

Reply:
Added a message to see the dedicated leader election section if the leader name is empty